core/storage/feedback/gae_models.py

```diff
@@ -525,10 +581,30 @@ def create(cls, user_id, thread_id):
             FeedbackThreadUserModel. The newly created FeedbackThreadUserModel
             instance.
         """
-        instance_id = cls.generate_full_id(user_id, thread_id)
-        new_instance = cls(id=instance_id, user_id=user_id, thread_id=thread_id)
-        new_instance.put()
-        return new_instance
+
+        return cls.create_multi(user_id, [thread_id])[0]
+
+    @classmethod
+    def create_multi(cls, user_id, thread_ids):
+        """Creates new FeedbackThreadUserModel instances for user_id for each
+        of the thread_ids.
+
+        Args:
+            user_id: str. The id of the user.
+            thread_ids: list(str). The ids of the threads.
+
+        Returns:
+            list(FeedbackThreadUserModel). The newly created
+            FeedbackThreadUserModel instances.
+        """
+        new_instances = []
+        for thread_id in thread_ids:
+            instance_id = cls.generate_full_id(user_id, thread_id)
+            new_instance = cls(
+                id=instance_id, user_id=user_id, thread_id=thread_id)
+            new_instances.append(new_instance)
+        GeneralFeedbackThreadUserModel.put_multi(new_instances)
```

Comment:
Perhaps add a newline above this one. (Conceptually, the block above creates the new instances, and this block onwards uses them.)